app/Platforms/IOSPlatform.php

```diff
@@ -30,4 +35,32 @@ public function getDownloadUrl(Application $application, Build $build, Cloud $st
         $plistUrl = route('applications.plist', [$application->slug, $this->getId()]);
         return 'itms-services://?action=download-manifest&url='.urlencode($plistUrl);
     }
+
+    /**
+     * @param  UploadedFile  $file
+     * @return string|null
+     */
+    public function getPackage(UploadedFile $file): ?string
+    {
+        $archive = new ZipArchive();
```

Comment:
Just curious, how big could be the file? is ok to unpack it in ram?

Reply:
The info.plist is just a couple of KBs